### DISCUSSION TITLE
I have a pixel 6a, just tried to install the most recent build, it failed.

### DISCUSSION THREAD

**Discussion by robertkraig** [NightscoutFoundation/xDrip#3727] [Bug reports]
When I download & install the newest version of xDrip+, it says: `This app was built for an older version of Android and doesn't include the latest privacy protections.

I'm on Pixel 6a w/ android build: AP2A.240905.003.F1 which is part of this release cycle I suppose: https://support.google.com/pixelphone/thread/294694412/google-pixel-update-september-2024?hl=en My intent is to use it for a Dexcom G6 (recent T2D diagnosis) so I wanted a better app to manage my stats.

**Comment by Navid200** [comment]
You can install xDrip as explained here: https://navid200.github.io/xDrip/docs/FAQ/UnsafeAppBlocked.html  

I have Pixel 6a and have been using it with xDrip and Dexcom G7 for a year.

xDrip is a single app that supports many sensors and many phones and many Android versions.  The price to pay is that there are some inconveniences like the message you have seen we have to put up with.
The alternative is to change xDrip not to support older versions of Android.  If we do that, we will take the ability to use xDrip from anyone who cannot afford to buy a new phone.  We will not do that.
